sqlup-mode.el

```diff
@@ -199,7 +199,7 @@ ANSI SQL keywords."
       'ansi))
 
 (defun sqlup-redis-mode-p ()
-  (eq major-mode #'redis-mode))
+  (eq major-mode 'redis-mode))
```

Comment:
Sort of: it is actually the symbol `redis-mode` rather than the function `redis-mode` that we want to compare with here (the major mode variable stores a symbol not a function), and `#'` is only for functions. Does that make sense? Not sure I'm explaining it very well.

Some potentially useful background info that might help if you don't know it already:

* emacs is a ["lisp-2"](http://stackoverflow.com/questions/4578574/what-is-the-difference-between-lisp-1-and-lisp-2) which means functions and variables are in separate namespaces, so they can have the same name.

* `#'` is just a compiler hint telling it to check that a function with that name exists at compilation time.